tests/integration/customizations/s3/test_plugin.py

```diff
@@ -466,8 +527,10 @@ def test_copy_large_file_signature_v4(self):
     def test_copy_metadata(self):
         # Copy the same style of parsing as the CLI session. This is needed
         # For comparing expires timestamp.
+        # TODO: james - do we need this?  seems like we shouldn't be doing
+        # this.
         add_scalar_parsers(self.session)
```

Comment:
Yeah I am not sure. Maybe because we create the session and a handler that is needed does not get added because it does not go through the entire CLI process?